concat-on-disk.ipynb

```diff
@@ -628,23 +654,23 @@
      "text": [
       "Dataset: dense 0\n",
       "Concatenating 6 files with sizes:\n",
-      "['668MiB', '827MiB', '668MiB', '920MiB', '875MiB', '668MiB']\n",
-      "Total size: 4630MiB\n",
+      "['668MiB', '668MiB', '919MiB', '668MiB', '932MiB', '932MiB']\n",
+      "Total size: 4789MiB\n",
       "Concatenation finished\n",
-      "Peak Memory: 2740 MiB\n",
+      "Peak Memory: 388 MiB\n",
       "--------------------------------------------------\n",
       "Dataset: dense 1\n",
       "Concatenating 6 files with sizes:\n",
-      "['912MiB', '823MiB', '668MiB', '668MiB', '668MiB', '864MiB']\n",
-      "Total size: 4606MiB\n",
+      "['859MiB', '668MiB', '885MiB', '668MiB', '892MiB', '668MiB']\n",
+      "Total size: 4641MiB\n",
       "Concatenation finished\n",
-      "Peak Memory: 3450 MiB\n",
+      "Peak Memory: 344 MiB\n",
       "--------------------------------------------------\n"
```

Comment:
These results are after doing the changes here: https://github.com/scverse/anndata/pull/1169

Reply:
As @ivirshup said: you also added `write_chunked`, are you sure the anndata changes are necessary?

Reply:
Yes I tried with it only first. If you try running the notebook on main branch of anndata you will get the old results.